packages/services/src/view-service/transaction-info.ts

```diff
@@ -3,16 +3,12 @@ import { servicesCtx } from '../ctx/prax';
 import { TransactionInfo } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/view/v1/view_pb';
 import { generateTransactionInfo } from '@penumbra-zone/wasm/transaction';
 import { fvkCtx } from '../ctx/full-viewing-key';
-import { Code, ConnectError } from '@connectrpc/connect';
 
 export const transactionInfo: Impl['transactionInfo'] = async function* (req, ctx) {
-  const services = ctx.values.get(servicesCtx);
+  const services = await ctx.values.get(servicesCtx);
   const { indexedDb } = await services.getWalletServices();
 
-  const fullViewingKey = ctx.values.get(fvkCtx);
-  if (!fullViewingKey) {
-    throw new ConnectError('Cannot access full viewing key', Code.Unauthenticated);
-  }
+  const fvk = ctx.values.get(fvkCtx);
```

Comment:
Is it possible to get the same await syntax as servicesCtx? Aka:

```
const fvk = await ctx.values.get(fvkCtx);
```

Reply:
this would complicate things on the configuration side.

i think making servicesCtx a bare promise was probably an incorrect choice, after experimenting with both techniques, so i'll align them both to be functions.